build_runner/lib/src/asset/cache.dart

```diff
@@ -4,11 +4,10 @@
 import 'dart:async';
```

Comment:
My intention was to keep package:build as small as possible until there is evidence it needs to grow. Since only build_runner stuff is using this so far I moved it there. If we come across something using that that isn't using other stuff from build_runner we can move it back and re-export it